addons/l10n_cl/views/report_invoice.xml

```diff
@@ -220,6 +219,57 @@
             <attribute name="t-call">l10n_cl.tax_totals_widget</attribute>
         </t>
 
+        <xpath expr="//div[hasclass('invoice_main')]" position="inside">
+            <t t-if="show_info_box">
+                <div class="row g-0" style="float: right;">
```

Comment:
in term of construction of code you have 2 choices either using different rows or one big row and put everything in it. That's the solution we chose even it's not the best in term of html it was the cleaner and compact code